README.md

```diff
@@ -1,75 +1,2 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
```

Comment:
- [ ] kindly add linters on your project. You can check this [repo](https://github.com/microverseinc/linters-config/tree/master/html-css) if you have any problem with installing linters or how to set them on your project.

Reply:
[OPTIONAL]
- [ ] Kindly, always make sure you explain why the changes were suggested :smile: 
`HINT: You can tell them what linters are for...`

Reply:
Thank you, Sir, next time I will 😄